2019-08-01 10:47:38.0133|INFO|school_diary.Controllers.AdminsController|Getting admin by admin ID, controller
2019-08-01 10:47:38.0732|INFO|school_diary.Services.AdminsService|Accssesing db over Admin rep, get admin by id
2019-08-01 10:47:47.6724|INFO|school_diary.Utilities.NLogExceptionLogger|Requested admin doesn't exists
2019-08-01 10:47:47.6724|INFO|school_diary.Utilities.NLogExceptionLogger|GET http://localhost:51055/api/admin/2a332618-defa-44c5-aefe-677f12312370
2019-08-01 10:47:47.6724|INFO|school_diary.Utilities.NLogExceptionLogger|school_diary.Models.DTOs.AdminDTOOut GetAdminById(System.String)
2019-08-01 10:57:57.0885|INFO|school_diary.Controllers.AdminsController|Getting admin by admin ID, controller
2019-08-01 10:57:57.1205|INFO|school_diary.Services.AdminsService|Accssesing db over Admin rep, get admin by id
2019-08-01 10:58:14.2146|INFO|school_diary.Utilities.NLogExceptionLogger|48
2019-08-01 10:58:14.2186|INFO|school_diary.Utilities.NLogExceptionLogger|GET http://localhost:51055/api/admin/2a332618-defa-44c5-aefe-677f12312370
2019-08-01 10:58:14.2186|INFO|school_diary.Utilities.NLogExceptionLogger|school_diary.Models.DTOs.AdminDTOOut GetAdminById(System.String)
2019-08-01 11:02:19.1764|INFO|school_diary.Controllers.AdminsController|Getting admin by admin ID, controller
2019-08-01 11:02:19.2124|INFO|school_diary.Services.AdminsService|Accssesing db over Admin rep, get admin by id
2019-08-01 11:02:28.6651|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception d:\Users\PC\source\repos\school_diary\school_diary\Services\AdminsService.cs, line which throw an exception 48
2019-08-01 11:02:28.6651|INFO|school_diary.Utilities.NLogExceptionLogger|GET http://localhost:51055/api/admin/2a332618-defa-44c5-aefe-677f12312370
2019-08-01 11:02:28.6651|INFO|school_diary.Utilities.NLogExceptionLogger|school_diary.Models.DTOs.AdminDTOOut GetAdminById(System.String)
2019-08-01 11:13:42.5384|INFO|school_diary.Controllers.AdminsController|Getting admin by admin ID, controller
2019-08-01 11:13:42.5744|INFO|school_diary.Services.AdminsService|Accssesing db over Admin rep, get admin by id
2019-08-01 11:13:53.1575|INFO|school_diary.Utilities.NLogExceptionLogger|Requested admin doesn't exists
2019-08-01 11:13:53.1575|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception d:\Users\PC\source\repos\school_diary\school_diary\Services\AdminsService.cs, line which throw an exception 48
2019-08-01 11:13:53.1575|INFO|school_diary.Utilities.NLogExceptionLogger|GET http://localhost:51055/api/admin/2a332618-defa-44c5-aefe-677f12312370
2019-08-01 11:13:53.1575|INFO|school_diary.Utilities.NLogExceptionLogger|school_diary.Models.DTOs.AdminDTOOut GetAdminById(System.String)
2019-08-01 11:15:21.9814|INFO|school_diary.Controllers.AdminsController|Getting admin by admin ID, controller
2019-08-01 11:15:22.0494|INFO|school_diary.Services.AdminsService|Accssesing db over Admin rep, get admin by id
2019-08-01 11:15:35.5361|INFO|school_diary.Utilities.NLogExceptionLogger|Exception message: Requested admin doesn't exists
2019-08-01 11:15:35.5361|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception d:\Users\PC\source\repos\school_diary\school_diary\Services\AdminsService.cs, line which throw an exception 48
2019-08-01 11:15:35.5361|INFO|school_diary.Utilities.NLogExceptionLogger|GET http://localhost:51055/api/admin/2a332618-defa-44c5-aefe-677f12312370
2019-08-01 11:15:35.5361|INFO|school_diary.Utilities.NLogExceptionLogger|school_diary.Models.DTOs.AdminDTOOut GetAdminById(System.String)
2019-08-01 11:28:44.6077|INFO|school_diary.Controllers.AdminsController|Getting admin by admin ID, controller
2019-08-01 11:28:44.6477|INFO|school_diary.Services.AdminsService|Accssesing db over Admin rep, get admin by id
2019-08-01 11:28:52.8596|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception d:\Users\PC\source\repos\school_diary\school_diary\Services\AdminsService.cs, line which throw an exception 48
2019-08-01 11:28:52.8596|INFO|school_diary.Utilities.NLogExceptionLogger|GET http://localhost:51055/api/admin/2a332618-defa-44c5-aefe-677f12312370
2019-08-01 11:28:52.8596|INFO|school_diary.Utilities.NLogExceptionLogger|school_diary.Models.DTOs.AdminDTOOut GetAdminById(System.String)
2019-08-01 11:28:52.8596|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: Requested admin doesn't exists
2019-08-01 11:36:09.0160|INFO|school_diary.Controllers.AppUsersController|Admin gets all users
2019-08-01 14:07:06.7403|INFO|school_diary.Controllers.AppUsersController|Admin gets all users, appuserscontroller
2019-08-01 14:07:06.7783|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get all users
2019-08-01 14:08:16.0897|INFO|school_diary.Controllers.AppUsersController|Admin gets user by username, appuserscontroller
2019-08-01 14:08:16.0917|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by username
2019-08-01 14:08:27.9330|INFO|school_diary.Controllers.AppUsersController|Admin gets user by username, appuserscontroller
2019-08-01 14:08:27.9330|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by username
2019-08-01 14:08:37.0709|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception d:\Users\PC\source\repos\school_diary\school_diary\Services\AppUsersService.cs, line which throw an exception 69
2019-08-01 14:08:37.4237|INFO|school_diary.Utilities.NLogExceptionLogger|GET http://localhost:51055/api/users/perateacher
2019-08-01 14:08:37.7916|INFO|school_diary.Utilities.NLogExceptionLogger|school_diary.Models.DTOs.AppUserDTOOut GetUserByUsername(System.String)
2019-08-01 14:08:38.2083|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: No user with username perateacher here!
2019-08-01 14:50:26.7841|INFO|school_diary.Controllers.AppUsersController|Admin gets user by id, appuserscontroller
2019-08-01 14:50:26.8261|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by id
2019-08-01 14:50:49.6363|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception d:\Users\PC\source\repos\school_diary\school_diary\Services\AppUsersService.cs, line which throw an exception 91
2019-08-01 14:50:49.8512|INFO|school_diary.Utilities.NLogExceptionLogger|GET http://localhost:51055/api/users/0526afee-c3c9-4256-9d66-c6d1539f12c9
2019-08-01 14:50:50.1070|INFO|school_diary.Utilities.NLogExceptionLogger|school_diary.Models.DTOs.AppUserDTOOut GetUserById(System.String)
2019-08-01 14:50:50.3839|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: No user with id 0526afee-c3c9-4256-9d66-c6d1539f12c9 here!
2019-08-01 14:51:31.9457|INFO|school_diary.Controllers.AppUsersController|Admin gets user by id, appuserscontroller
2019-08-01 14:51:31.9457|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by id
2019-08-01 15:19:04.6715|INFO|school_diary.Controllers.AppUsersController|Admin gets user by name petar and lastname ignjatov, appuserscontroller
2019-08-01 15:19:04.7065|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by name and lastname
2019-08-01 15:20:13.1583|INFO|school_diary.Controllers.AppUsersController|Admin gets user by lastname petar, appuserscontroller
2019-08-01 15:20:13.1583|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by lastname
2019-08-01 15:20:18.3825|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception d:\Users\PC\source\repos\school_diary\school_diary\Services\AppUsersService.cs, line which throw an exception 115
2019-08-01 15:20:18.6514|INFO|school_diary.Utilities.NLogExceptionLogger|GET http://localhost:51055/api/users/by-lastname/petar
2019-08-01 15:20:18.8995|INFO|school_diary.Utilities.NLogExceptionLogger|System.Collections.Generic.IEnumerable`1[school_diary.Models.DTOs.AppUserDTOOut] GetUserBySurname(System.String)
2019-08-01 15:20:19.4367|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: No users with name petar here!!!
2019-08-01 15:20:56.1084|INFO|school_diary.Controllers.AppUsersController|Admin gets user by name petar, appuserscontroller
2019-08-01 15:20:56.1084|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by name
2019-08-01 15:27:30.7735|INFO|school_diary.Controllers.AppUsersController|Deleting user, app user controller
2019-08-01 15:27:30.8145|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by username
2019-08-01 15:28:17.7505|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception , line which throw an exception 0
2019-08-01 15:28:17.9803|INFO|school_diary.Utilities.NLogExceptionLogger|DELETE http://localhost:51055/api/users/aleksastudent
2019-08-01 15:28:18.2282|INFO|school_diary.Utilities.NLogExceptionLogger|T NotNull[T](T, System.String)
2019-08-01 15:28:18.7012|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: Value cannot be null.
Parameter name: entity
2019-08-01 15:30:26.2198|INFO|school_diary.Controllers.AppUsersController|Deleting user, app user controller
2019-08-01 15:30:39.0225|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by username
2019-08-01 15:31:22.0522|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception , line which throw an exception 0
2019-08-01 15:31:22.2271|INFO|school_diary.Utilities.NLogExceptionLogger|DELETE http://localhost:51055/api/users/aleksastudent
2019-08-01 15:31:22.6721|INFO|school_diary.Utilities.NLogExceptionLogger|T NotNull[T](T, System.String)
2019-08-01 15:31:23.0855|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: Value cannot be null.
Parameter name: entity
2019-08-01 15:32:54.9285|INFO|school_diary.Controllers.AppUsersController|Admin gets user by name aleksastudent, appuserscontroller
2019-08-01 15:32:54.9285|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by name
2019-08-01 15:33:07.7403|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception d:\Users\PC\source\repos\school_diary\school_diary\Services\AppUsersService.cs, line which throw an exception 91
2019-08-01 15:33:08.0593|INFO|school_diary.Utilities.NLogExceptionLogger|GET http://localhost:51055/api/users/by-name/aleksastudent
2019-08-01 15:33:08.2691|INFO|school_diary.Utilities.NLogExceptionLogger|System.Collections.Generic.IEnumerable`1[school_diary.Models.DTOs.AppUserDTOOut] GetUserByName(System.String)
2019-08-01 15:33:08.5801|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: No users with name aleksastudent here!!!
2019-08-01 15:40:26.9294|INFO|school_diary.Controllers.AppUsersController|Admin gets user by username, appuserscontroller
2019-08-01 15:40:26.9294|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by username
2019-08-01 15:41:40.2608|INFO|school_diary.Controllers.AppUsersController|Deleting user, app user controller
2019-08-01 15:41:40.2608|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by username
2019-08-01 15:41:50.1315|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception , line which throw an exception 0
2019-08-01 15:41:50.3953|INFO|school_diary.Utilities.NLogExceptionLogger|DELETE http://localhost:51055/api/users/aleksastudent
2019-08-01 15:41:50.5412|INFO|school_diary.Utilities.NLogExceptionLogger|T NotNull[T](T, System.String)
2019-08-01 15:41:51.0374|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: Value cannot be null.
Parameter name: entity
2019-08-01 15:45:59.7977|INFO|school_diary.Controllers.AppUsersController|Deleting user, app user controller
2019-08-01 15:46:01.0812|INFO|school_diary.Services.AppUsersService|Accessing db over users repository, get user by username
2019-08-01 15:46:09.9779|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception , line which throw an exception 0
2019-08-01 15:46:10.2138|INFO|school_diary.Utilities.NLogExceptionLogger|DELETE http://localhost:51055/api/users/aleksastudent
2019-08-01 15:46:10.4467|INFO|school_diary.Utilities.NLogExceptionLogger|T NotNull[T](T, System.String)
2019-08-01 15:46:11.1982|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: Value cannot be null.
Parameter name: entity
2019-08-01 23:33:05.7553|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception , line which throw an exception 0
2019-08-01 23:33:05.9233|INFO|school_diary.Utilities.NLogExceptionLogger|DELETE http://localhost:51055/api/students/aleksastudent
2019-08-01 23:33:06.2395|INFO|school_diary.Utilities.NLogExceptionLogger|T NotNull[T](T, System.String)
2019-08-01 23:33:07.0279|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: Value cannot be null.
Parameter name: entity
2019-08-01 23:45:54.5053|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception d:\Users\PC\source\repos\school_diary\school_diary\Services\StudentsService.cs, line which throw an exception 51
2019-08-01 23:45:54.7941|INFO|school_diary.Utilities.NLogExceptionLogger|DELETE http://localhost:51055/api/students/009e1361-e6b4-400c-a955-f3f32e9d37ff
2019-08-01 23:45:55.0660|INFO|school_diary.Utilities.NLogExceptionLogger|school_diary.Models.Student GetStudentByUserName(System.String)
2019-08-01 23:45:55.3619|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: Student with username 009e1361-e6b4-400c-a955-f3f32e9d37ff does not exists here
2019-08-01 23:47:21.6107|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception d:\Users\PC\source\repos\school_diary\school_diary\Services\StudentsService.cs, line which throw an exception 51
2019-08-01 23:47:21.6107|INFO|school_diary.Utilities.NLogExceptionLogger|DELETE http://localhost:51055/api/students/0db74eca-8103-4e24-b9fb-f82d4c80bc30
2019-08-01 23:47:21.6107|INFO|school_diary.Utilities.NLogExceptionLogger|school_diary.Models.Student GetStudentByUserName(System.String)
2019-08-01 23:47:21.6217|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: Student with username 0db74eca-8103-4e24-b9fb-f82d4c80bc30 does not exists here
2019-08-01 23:50:23.5195|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception d:\Users\PC\source\repos\school_diary\school_diary\Services\StudentsService.cs, line which throw an exception 61
2019-08-01 23:50:23.8674|INFO|school_diary.Utilities.NLogExceptionLogger|DELETE http://localhost:51055/api/students/0db74eca-8103-4e24-b9fb-f82d4c80bc30
2019-08-01 23:50:24.2513|INFO|school_diary.Utilities.NLogExceptionLogger|school_diary.Models.DTOs.StudentDTOOut GetStudentById(System.String)
2019-08-01 23:50:24.5392|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: Student with ID 0db74eca-8103-4e24-b9fb-f82d4c80bc30 does not exists here
2019-08-01 23:55:25.6466|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception , line which throw an exception 0
2019-08-01 23:55:25.6546|INFO|school_diary.Utilities.NLogExceptionLogger|DELETE http://localhost:51055/api/students/0617a92f-f577-489b-8acf-ded56c26d628
2019-08-01 23:55:25.6546|INFO|school_diary.Utilities.NLogExceptionLogger|Int32 SaveChanges()
2019-08-01 23:55:25.6546|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: An error occurred while saving entities that do not expose foreign key properties for their relationships. The EntityEntries property will return null because a single entity cannot be identified as the source of the exception. Handling of exceptions while saving can be made easier by exposing foreign key properties in your entity types. See the InnerException for details.
2019-08-01 23:56:46.8743|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception d:\Users\PC\source\repos\school_diary\school_diary\Services\StudentsService.cs, line which throw an exception 61
2019-08-01 23:56:46.9223|INFO|school_diary.Utilities.NLogExceptionLogger|DELETE http://localhost:51055/api/students/0617a92f-f577-489b-8acf-ded56c26d628
2019-08-01 23:56:46.9223|INFO|school_diary.Utilities.NLogExceptionLogger|school_diary.Models.DTOs.StudentDTOOut GetStudentById(System.String)
2019-08-01 23:56:46.9423|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: Student with ID 0617a92f-f577-489b-8acf-ded56c26d628 does not exists here
2019-08-01 23:58:01.4752|INFO|school_diary.Utilities.NLogExceptionLogger|File throwing an exception , line which throw an exception 0
2019-08-01 23:58:01.6592|INFO|school_diary.Utilities.NLogExceptionLogger|DELETE http://localhost:51055/api/students/82eae9b8-acf1-432e-ab6b-ff13c3401fe9
2019-08-01 23:58:01.8431|INFO|school_diary.Utilities.NLogExceptionLogger|Int32 SaveChanges()
2019-08-01 23:58:02.0271|ERROR|school_diary.Utilities.NLogExceptionLogger|Exception message: An error occurred while saving entities that do not expose foreign key properties for their relationships. The EntityEntries property will return null because a single entity cannot be identified as the source of the exception. Handling of exceptions while saving can be made easier by exposing foreign key properties in your entity types. See the InnerException for details.
